test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -66,6 +66,27 @@ const filename = tmpdir.resolve('sync-write-stream.txt');
   assert.strictEqual(stream.fd, null);
 }
 
+// Verify behavior of destroy() when already destroy()ed
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd);
+
+  stream.on('close', common.mustCall());
+  assert.strictEqual(stream.destroy(), stream);
+  assert.strictEqual(stream.destroy(), stream);
+}
+
```

Comment:
```suggestion
```

This is the standard stream behavior. I would not add it because it is tested in other places but I also will not block if you want to keep it.